index.html

```diff
@@ -48,9 +56,10 @@ <h1 class="title">I'm Shaheda Glad to see you!
     </div>
 
     <section class="works-section">
-        <ul class="card">
+        <ul class="card1">
             <li class="project">
-                <img src="src/SnapshootPortfolio.png" alt="Snapshot">
+                <img class="mobileimg" src="src/SnapshootPortfolio.png" alt="Snapshot">
+                <img class="image" src="desk/SnapshootPortfolio4.png" alt="Snapshot">
```

Comment:
- [ ] Kindly your mobile version is good just a small problem is that there are overflows in your page because this image if you give it width (recommended with %) it will solve the issue. Kindly review your mobile version too because sometimes when making the desktop version the mobile version broke in some situations.

![image](https://user-images.githubusercontent.com/35535130/155709434-5ff2c04f-f710-4c0b-8128-cbfa53566e60.png)